content/academy/node_js/parsing_compressed_sitemaps.md

```diff
@@ -0,0 +1,221 @@
+---
+title: How to parse compressed sitemaps
```

Comment:
This might be solved in Crawlee already

Reply:
yeah i believe we already handle compression in the sitemap helper, cc @janbuchar